Source/JavaScriptCore/inspector/InjectedScriptBase.cpp

```diff
@@ -95,7 +95,7 @@ Ref<JSON::Value> InjectedScriptBase::makeCall(Deprecated::ScriptFunctionCall& fu
 
     auto resultJSONValue = toInspectorValue(globalObject, value);
     if (!resultJSONValue)
-        return JSON::Value::create(makeString("Object has too long reference chain (must not be longer than ", JSON::Value::maxDepth, ')'));
+        return JSON::Value::create(makeString("Object has too long reference chain (must not be longer than ", unsigned(JSON::Value::maxDepth), ')'));
```

Comment:
I can never remember what the style guide says for this

Reply:
I can do `static_cast`.